resources/sample_vaults/Tasks-Demo/.obsidian/appearance.json

```diff
@@ -1,5 +1,5 @@
 {
-  "theme": "moonstone",
+  "theme": "system",
```

Comment:
Got Blinded multiple Times, soo :)

Reply:
> Got Blinded multiple Times, soo :)

I could have sworn that I had previously done that - very happy with this change!